laptop-specific/battery.md

```diff
@@ -1,6 +1,38 @@
-# Battery Patching
+# Battery Status
 
-While at the moment battery patching is not something covered in this guide, we'll at least point you to some helpful resources and add some useful notes for when using them in OpenCore.
+While previously you may have had to create DSDT patches/SSDT hot-patches in order to get working battery status in macOS, you longer have to thanks to the [ECEnabler](https://github.com/1Revenger1/ECEnabler/releases/latest) kext. In the extraordinary case that you still have to create DSDT patches/SSDT hot-patches (e.g. for dual batteries), we have left some resources linked below.
+
+* If battery status is not working even with ECEnabler, make sure you have the [SMCBatteryManager](https://github.com/Acidanthera/VirtualSMC/releases/latest) VirtualSMC plugin enabled in your OpenCore configuration.
+
+* Please note that certain devices, such as the Surface 3, Surface Pro 5, Surface Book 2, and Surface Laptop (and all subsequent Surface devices), use proprietary Embedded Controllers (or other similar hardware) instead of standard ACPI battery devices and OperationRegion fields, and thus without device-specific kexts, battery status cannot work.
```

Comment:
```suggestion
* Certain devices, such as the Surface 3, Surface Pro 5, Surface Book 2, and Surface Laptop (and all subsequent Surface devices), use proprietary Embedded Controllers (or other similar hardware) instead of standard ACPI battery devices and OperationRegion fields, and thus without device-specific kexts, battery status cannot work.
```

maybe make it a tip/warning/whatever?